modin/pandas/test/test_io.py

```diff
@@ -175,7 +161,16 @@ def assert_files_eq(path1, path2):
 
 def teardown_test_file(test_path):
     if os.path.exists(test_path):
-        os.remove(test_path)
+        # PermissionError can occure because of issue #2533
```

Comment:
Does this occur when files with the same names are created in parallel in different processes?

Reply:
No, it affects both parallel and sequential execution modes.